frontend/src/app/testQueue/addToQueue/SearchInput.tsx

```diff
@@ -76,6 +77,7 @@ const SearchInput = ({
               disabled={disabled}
               onClick={onSearchClick}
             >
+              <IconSearch size={3} />
               <span className="usa-sr-only">Search</span>
```

Comment:
I wonder if it's possible to pass an `alt` or `alt-text` prop to `IconSearch` instead of using the existing span? Kind of like the [uswds design system example](https://designsystem.digital.gov/components/search/?search=):
```
<button class="usa-button" type="submit">
      <img
        src="/assets/img/usa-icons-bg/search--white.svg"
        class="usa-search__submit-icon"
        alt="Search"
      />
    </button>
    ```

Totally fine if not, this works, just thought it might clean it up.

Reply:
@emyl3 @georgehager 

I think just using image and that class seems to fixes the centered thing! 
![image](https://user-images.githubusercontent.com/10108172/193833875-3301d30c-0260-4145-bb66-3d85dc4caab4.png)